Common/src/main/java/jeresources/compatibility/minecraft/MinecraftCompat.java

```diff
@@ -102,20 +102,20 @@ private void registerVanillaPlants() {
         PlantDrop poisonous = new PlantDrop(new ItemStack(Items.POISONOUS_POTATO), 0.02F);
         registerPlant(new PlantEntry((PotatoBlock) Blocks.POTATOES, potato, poisonous));
 
-        //Carrot
+        // Carrot
```

Comment:
can you undo just adding the spaces? I don't really see the need for change in this file just for formatting of a comment

Reply:
:)